### ISSUE TITLE
Thorify web3-gear web3 does not work together at all

### ISSUE THREAD

**Issue by sangduongvan** [vechain/thorify#78]
Hi there,

please note that this is an issue tracker reserved for bug reports and feature requests.

For general questions please use the gitter channel.

#### Environment information
1. Thor solo mode
Starting Thor solo/v1.0.5-758fa21-dev/darwin/go1.11.4
    Network     [ 0x00000000973ceb7f343a58b08f0693d6701a5fd354ff73d7058af3fba222aea4 devnet ]
    Best block  [ 0x00000000973ceb7f343a58b08f0693d6701a5fd354ff73d7058af3fba222aea4 #0 @2018-05-15 23:00:00 +0700 +07 ]
    Forks       [ FTRL: #0 ]
    Data dir    [ Memory ]
    API portal  [ http://127.0.0.1:8669/ ]

2. web3-gear
Web3-Gear/1.1.1/darwin/python3.7.1
Listening on 127.0.0.1:8545

>>>cat package.json
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "test.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "thorify": "^1.0.6",
    "web3": "1.0.0-beta.35"
  }
}
Test code:
```
cat test.js
const thorify = require("thorify").thorify;
const Web3 = require("web3");

//let web3;
const network = "http://127.0.0.1:8545";
let web3 = thorify(new Web3(), network);

web3.eth.getBlock("latest").then(res => console.log(res));
```
#### Actual behaviour
Result ```
node test.js
{ error: { code: -32700, message: 'Parse error' },
  id: null,
  jsonrpc: '2.0',
  gasLimit: undefined,
  gasUsed: undefined,
  size: undefined,
  timestamp: undefined,
  number: undefined }
```

#### Steps to reproduce the behaviour


#### Stacktrace

````
[stacktrace]
````

**Comment by libotony** [comment]
They are designed not to work together. See [doc](https://github.com/vechain/thorify#web3-gear-vs-thorify)

**Comment by sangduongvan** [comment]
ah o, let me try again

**Comment by sangduongvan** [comment]
It's ok now, thank you
